test/functional/apps/visualize/_tsvb_time_series.ts

```diff
@@ -86,6 +86,33 @@ export default function({ getPageObjects, getService }: FtrProviderContext) {
         const actualCount = await visualBuilder.getRhythmChartLegendValue();
         expect(actualCount).to.be(expectedLegendValue);
       });
+
+      it('should show the correct count in the legend with percent formatter', async () => {
+        const expectedLegendValue = '15,600%';
+
+        await visualBuilder.clickSeriesOption();
+        await visualBuilder.changeDataFormatter('Percent');
+        const actualCount = await visualBuilder.getRhythmChartLegendValue();
+        expect(actualCount).to.be(expectedLegendValue);
+      });
+
+      it('should show the correct count in the legend with bytes formatter', async () => {
+        const expectedLegendValue = '156B';
+
+        await visualBuilder.clickSeriesOption();
+        await visualBuilder.changeDataFormatter('Bytes');
+        const actualCount = await visualBuilder.getRhythmChartLegendValue();
+        expect(actualCount).to.be(expectedLegendValue);
+      });
+
+      it('should show the correct count in the legend with duration formatter', async () => {
```

Comment:
@dmlemeshko @vitalics  Could you please add tests for #35813.

Reply:
complex duration formatter tests will be in the next PR